src/intpack.f90

```diff
@@ -983,7 +983,7 @@ pure subroutine opac3(l,m,n,ga,v,d)
    !           electronic part of charge density
    implicit none
    integer,intent(in)  :: l,m,n
-   real(wp), intent(in)  :: ga,d(1)
+   real(wp), intent(in)  :: ga,d(3)
```

Comment:
`d(1)` was used as assumed shape array here -> `d(*)`. `d` is indeed a three element vector.

Reply:
Should I convert it to `d(*)`, or my changes accepted? 
I changed it for fixing the following warning:
```
/dev/shm/xtb/src/intpack.f90:992:15:

  991 |    do i=1,3
      |           2    
  992 |       dd(i)=-d(i)
      |               1
Warning: Array reference at (1) out of bounds (3 > 1) in loop beginning at (2)
```

Reply:
It is fine that way, I was just commenting on the original intent such that we can compare notes here.